liquidprompt

```diff
@@ -311,6 +309,13 @@ __lp_source_config() {
     LP_MARK_DIRSTACK="${LP_MARK_DIRSTACK:-"⚞"}"
     LP_MARK_SHLVL="${LP_MARK_SHLVL:-"└"}"
     LP_MARK_WIFI="${LP_MARK_WIFI:-"📶"}"
+    LP_MARK_DEV_OPEN="${LP_MARK_DEV_OPEN:-"<"}"
+    LP_MARK_DEV_CLOSE="${LP_MARK_DEV_CLOSE:-">"}"
```

Comment:
Breaking out these marks is the right call.

I think that they should stick with the previous default of `[...]` though, I don't see a good reason to change it.

Reply:
It is faster to visually parse sections with different types of brackets.

Squared brackets are already used for the central part of the prompt. Parentheses are already used in the VCS section.

That leaves us with either `<…|…>` or `{…|…}`. I picked at random, but curly braces seems also good, especially for people being used to maths. I tested both for some time, and I find that `<…|…>` seems faster to spot and easier to parse, as it's less similar to the other sections.

Reply:
That's sound logic, you have convinced me.

We'll want to make sure we add that reasoning to the changelog and release notes.